src/Umbraco.Tests/PropertyEditors/MultiValuePropertyEditorTests.cs

```diff
@@ -88,6 +99,22 @@ public void DropDownValueEditor_Format_Data_For_Cache()
         [Test]
         public void DropDownPreValueEditor_Format_Data_For_Editor()
         {
+            // editor wants ApplicationContext.Current.Services.TextService
+            // (that should be fixed with proper injection)
+            var logger = Mock.Of<ILogger>();
+            var textService = new Mock<ILocalizedTextService>();
+            textService.Setup(x => x.Localize(It.IsAny<string>(), It.IsAny<CultureInfo>(), It.IsAny<IDictionary<string, string>>())).Returns("blah");
+            var appContext = new ApplicationContext(
+                new DatabaseContext(Mock.Of<IDatabaseFactory>(), logger, new SqlSyntaxProviders(Enumerable.Empty<ISqlSyntaxProvider>())),
+                new ServiceContext(
+                    localizedTextService: textService.Object
+                ),
+                Mock.Of<CacheHelper>(),
+                new ProfilingLogger(logger, Mock.Of<IProfiler>()))
+            {
+                IsReady = true
+            };
+            ApplicationContext.Current = appContext;
```

Comment:
Ahh, found out why, i'll make it injected. Property Editors (and any resolved class) support ctor injection now